docs/how-to/configure.md

```diff
@@ -103,9 +93,36 @@ Grafana’s [anonymous usage statistics](https://grafana.com/docs/grafana/latest
 
 As well as via the *PMM Settings* page, you can also disable telemetry with the `-e DISABLE_TELEMETRY=1` option in your docker run statement for the PMM Server.
 
+The following are some of the commonly used environment varibles for telemetry:
+
+```sh
+DISABLE_TELEMETRY=true
+```
+      Disable the telemetry feature
+
+```sh
+PERCONA_TEST_TELEMETRY_DISABLE_SEND=true
+```
+    Fetch metrics, but do not send to Platform
+
+```sh
+PERCONA_TEST_TELEMETRY_DISABLE_START_DELAY=true  
+```
+    Fetch metrics while PMM is starting
+
+```sh
+PERCONA_TEST_TELEMETRY_INTERVAL=24h 
+```
+    Change interval for telemetry iteration
+
+
+For information on the various config parameters for telemetry, see the [config file](https://github.com/percona/pmm/blob/main/managed/services/telemetry/config.default.yml).
+
+
 !!! note alert alert-primary ""
 
-    Telemetry is sent straight away; the 24 hour grace period is not honored.
+    PMM retains the telemetry data by default for 30 days. However, this can be modified by an admin from the settings page.
```

Comment:
I'm not able to get the context here. if this is to PERCONA_TEST_*  this  need to be clarified, but I would avoid PERCONA_TEST_ in official doc at all

Reply:
Wait, 30 days relate to metrics in general. 
We don't have such constrains for Telemetry data at all.

Reply:
This is why I think this line should stay as is. No changes are needed here.